python/tvm/contrib/cudnn.py

```diff
@@ -413,3 +411,324 @@ def conv2d_forward(x,
             ins[1],
             outs[0],
             conv_dtype), name="y")
+
+
+def conv3d_w_shape(in_channel,
```

Comment:
let's remove this function. This is trivial and only meaningful for NCHW layout. Also remove conv2d_w_shape